conftest.py

```diff
@@ -1,15 +1,37 @@
-# This file exists just to allow ignoring warnings without test collection failing on CI
-# TODO: Fix that
+# This file exists
+# 1. to allow ignoring warnings without test collection failing on CI
+# 2. as a pytest plugin/config that applies to doctests as well
+# TODO: Fix that, e.g. with the `pytest -p anndata.testing._pytest` pattern.
+
+from pathlib import Path
+
 import pytest
 
+from anndata.compat import chdir
+
+
+doctest_marker = pytest.mark.usefixtures("doctest_env")
+
+
+@pytest.fixture
+def doctest_env(cache: pytest.Cache, tmp_path: Path) -> None:
+    from scanpy import settings
+
+    old_dd, settings.datasetdir = settings.datasetdir, cache.mkdir("scanpy-data")
```

Comment:
Where does the cached `scanpy-data` get made?

Reply:
in .pytest-cache